### ISSUE TITLE
Error: Unknow type UpdateStrategy

### ISSUE THREAD

**Issue by ghost** [clementd64/tachiyomi-backup-models#2]
Thanks for creating this project! On deno 1.27.2 with a backup created just now on Tachiyomi 0.14.2, I get 

```
error: Uncaught (in promise) Error: Unknow type UpdateStrategy
    throw new Error(`Unknow type ${type}`);
          ^
    at Protobuf.type (file:///Users/sam/Downloads/tachiyomi-backup-models/src/generator.ts:18:11)
    at Protobuf.build (file:///Users/sam/Downloads/tachiyomi-backup-models/src/protobuf.ts:22:97)
    at Protobuf.process (file:///Users/sam/Downloads/tachiyomi-backup-models/src/generator.ts:26:25)
    at main (file:///Users/sam/Downloads/tachiyomi-backup-models/src/cli.ts:21:10)
    at async file:///Users/sam/Downloads/tachiyomi-backup-models/mod.ts:10:3
```

This happens both with local (which btw have moved to `tachiyomi/app/src/main/java/eu/kanade/tachiyomi/data/backup/models/`, it seems) and with auto-fetching of modules.

I'd really appreciate if you could help out with this, as I have literally no knowled of protobuf at all. Cheers!

**Comment by clementd64** [comment]
Thanks for reporting this issue.

Their added a new emun in the backup. It's an internal enum which is not part of the backup specific type, so not parsed.
A temporary fix was done on 1ed53dfcd9804edee04a131835cd80f211377cd0

**Comment by ghost** [comment]
Works, thanks a bunch!
